keyboards/snampad/readme.md

```diff
@@ -0,0 +1,15 @@
+# snampad
+
+![snampad](imgur.com image replace me!)
```

Comment:
You should include a direct image URL here (photo or render), if you have one. Otherwise, this line should be deleted.